pkg/feature/cert.go

```diff
@@ -126,3 +123,114 @@ func generateCertificate(addr string) ([]byte, []byte, error) {
 
 	return certBuffer.Bytes(), keyBuffer.Bytes(), nil
 }
+
+// GetDefaultIngressCertificate copies ingress cert secrets from openshift-ingress ns to given namespace.
```

Comment:
so we have two different ns: openshift-ingress is only to get secret, and openshift-ingress-operator is for ingresscontroller CR?